examples/timeseries/TS9_create_cattimeseries_from_timeseries.py

```diff
@@ -0,0 +1,35 @@
+# Standard library imports
+# /
+
+# Third party imports
+# /
+
+# Local application imports
+from scifin import timeseries as ts
+
+# Build a time series from a CSV file online
+ts1 = ts.build_from_csv(filepath_or_buffer='https://raw.githubusercontent.com/selva86/datasets/master/a10.csv',
+                        parse_dates=['date'], index_col='date', unit="Number of sales", name="Sales_TimeSeries")
+
+# Define min and max values of a range
+range_min = 10
+range_max = 20
+
+# Create a DataFrame with categories
+cts2_idx = ts1.data.index
+cts2_vals = []
+for x in range(ts1.nvalues):
+    if ts1.data.values[x] >= range_min and ts1.data.values[x] <= range_max:
```

Comment:
again, the two conditionals can be combined into one

Reply:
Thx.